modules/geo-layers/src/mvt-layer/mvt-layer.js

```diff
@@ -228,9 +229,26 @@ export default class MVTLayer extends TileLayer {
     }
   }
 
-  _onViewportLoad() {
-    super._onViewportLoad();
-    this._onViewportChange();
+  _setWGS84PropertyForTiles() {
+    const propName = 'dataInWGS84';
+    const {tileset} = this.state;
+
+    tileset.selectedTiles.forEach(tile => {
+      if (!tile.hasOwnProperty(propName)) {
+        // eslint-disable-next-line accessor-pairs
+        Object.defineProperty(tile, propName, {
+          get: () => {
+            if (!tile.contentWGS84) {
+              // Create a cache to transform only once
+              tile.contentWGS84 = tile.content.map(feature =>
```

Comment:
content can be `null` (still loading or encountered an error)

Reply:
- Still not checking `tile.content` before calling `map`
- Add an underscore before the name of internal cache

Reply:
Sorry I missed this comment

Reply:
Done